clover-conversion/Clover-config.md

```diff
@@ -194,6 +196,7 @@ So with the transition from Clover to OpenCore we should start removing unneeded
 **USB**:
 
 * FixOwnership: `UEFI -> Quirk -> ReleaseUsbOwnership`
+  * It is highly recommended to enable `XHCI Hand-off` in your BIOS. More details [here](https://https://dortania.github.io/OpenCore-Post-Install/misc/xhci-handoff.html)
```

Comment:
Some firmware vendors, such as Dell, may not have this option explicitly in the BIOS GUI. Despite leaving this quirk enabled shouldn't create any issue so far, further steps are required to enable `XHCI Hand-off`. The steps are similar to `CFG Lock` unlocking. I'll draft another PR based on dortania/OpenCore-Post-Install@d85b635